From b4f903cb109b455d7a01f24794c22b35f103e2a4 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Thu, 6 Aug 2020 14:38:57 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1126: add version in model name on evb
 board

uboot will parse and print model name, add version information
for debug.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I380548037a56321f78d1c24e1877f06a098330e0
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts | 2 +-
 arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
index cefa99768989..34d06abb1bea 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
@@ -8,7 +8,7 @@
 #include "rv11xx-evb-v10.dtsi"
 
 / {
-	model = "Rockchip RV1126 EVB DDR3 Board";
+	model = "Rockchip RV1126 EVB DDR3 V10 Board";
 	compatible = "rockchip,rv1126-evb-ddr3-v10", "rockchip,rv1126";
 
 	chosen {
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts
index 28d50b2bb93f..1b74d59eb1d8 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts
@@ -6,7 +6,7 @@
 #include "rv1126.dtsi"
 #include "rv11xx-evb-v12.dtsi"
 / {
-	model = "Rockchip RV1126 EVB DDR3 Board";
+	model = "Rockchip RV1126 EVB DDR3 V12 Board";
 	compatible = "rockchip,rv1126-evb-ddr3-v12", "rockchip,rv1126";
 
 	chosen {
-- 
2.35.3

